src/types/onyx/ApprovalWorkflow.ts

```diff
@@ -0,0 +1,84 @@
+import type {AvatarSource} from '@libs/UserUtils';
+
+/**
+ * Approver in the approval workflow
+ */
+type Approver = {
+    /**
+     * Email of the approver
+     */
+    email: string;
+
+    /**
+     * Email of the user this user forwards all approved reports to
+     */
+    forwardsTo?: string;
+
+    /**
+     * Avatar URL of the current user from their personal details
+     */
+    avatar?: AvatarSource;
+
+    /**
+     * Display name of the current user from their personal details
+     */
+    displayName?: string;
+
+    /**
+     * Is this approver in more than one workflow
+     */
+    isInMultipleWorkflows: boolean;
+
+    /**
+     * Is this approver in a circular reference
+     */
+    isCircularReference?: boolean;
+};
+
+/**
+ *
```

Comment:
You can probably just delete this comment block

Reply:
I don't think I can remove it, it comes back on save hahah